pygments/lexers/php.py

```diff
@@ -221,7 +221,7 @@ class PhpLexer(RegexLexer):
              r'array|E_ALL|NULL|final|php_user_filter|interface|'
              r'implements|public|private|protected|abstract|clone|try|'
              r'catch|throw|this|use|namespace|trait|yield|'
-             r'finally)\b', Keyword),
+             r'finally|match)\b', Keyword),
```

Comment:
I'm pretty sure `readonly` is in the right place, however there's two places where `switch` is defined (`Keyword` and `Keyword.Declaration`). Is there a way I can test this to make sure it works? Match works just like switch:

```php
switch ($var) {
  case 'a':
    $result = "some value";
    break;
  case 'b':
    $result = "some other value";
    break;
}

// can be replaced with

$result = match ($var) {
  'a' => "some value",
  'b' => "some other value",
};
```

Reply:
I don't see the other place? In any case, this LGTM.